packages/dbml-core/types/model_structure/database.d.ts

```diff
@@ -16,6 +16,19 @@ export interface Project {
     database_type: string;
     name: string;
 }
+
+export interface Records {
+    [tableSchemaName: string]: {
+        schemaName: string | undefined;
+        tableName: string;
+        columns: string[];
+        values: {
+            value: any;
+            type: string;
+        }[];
+    }
+}
```

Comment:
I prefer to break down type like this, reason being that if I ever need to borrow individual Record type elsewhere, I don't have to redefine it or use Records[string] as a workaround
```suggestion
export interface Record {
    schemaName: ...
    tableName: ...
    ...
}

export interface Records { [...]: Record }
```

Reply:
ye agree a, I updated in https://github.com/holistics/dbml/pull/690/commits/8177973ae2c8207dd2823c753e1b1e8a7f09d4a1